zcash_client_sqlite/src/wallet/sapling.rs

```diff
@@ -318,11 +318,11 @@ pub(crate) fn get_sapling_nullifiers<P>(
 }
 
 /// Returns the nullifiers for the notes that this wallet is tracking.
-pub(crate) fn get_all_sapling_nullifiers<P>(
-    wdb: &WalletDb<P>,
+pub(crate) fn get_all_sapling_nullifiers(
+    conn: &Connection,
 ) -> Result<Vec<(AccountId, Nullifier)>, SqliteClientError> {
     // Get the nullifiers for the notes we are tracking
-    let mut stmt_fetch_nullifiers = wdb.conn.prepare(
+    let mut stmt_fetch_nullifiers = conn.prepare(
```

Comment:
Why `prepare` rather than `prepare_cached` here?

Reply:
The only cases where I changed to use `prepare_cached` were those where we were previously caching prepared statements. It might be reasonable to cache here as well; I haven't looked closely to determine whether it would make sense to do so.

Reply:
There appears to be no down-side. This isn't a method that is only used at startup or anything like that.